frontend/class-frontend.php

```diff
@@ -1638,12 +1638,19 @@ function rss_replace_vars( $content ) {
 		$post_link      = '<a ' . $no_follow_attr . 'href="' . esc_url( get_permalink() ) . '">' . get_the_title() . '</a>';
 		$blog_link      = '<a ' . $no_follow_attr . 'href="' . esc_url( get_bloginfo( 'url' ) ) . '">' . get_bloginfo( 'name' ) . '</a>';
 		$blog_desc_link = '<a ' . $no_follow_attr . 'href="' . esc_url( get_bloginfo( 'url' ) ) . '">' . get_bloginfo( 'name' ) . ' - ' . strip_tags( get_bloginfo( 'description' ) ) . '</a>';
+		//reduce excerpt to 15 characters by using regex & end with whole word
+		$body= get_the_excerpt();
+		$line= $body;
+		if (preg_match('/^.{1,15}\b/s', $body, $match)) {
+			$excerpt_link   = '<a ' . $no_follow_attr . 'href="' . esc_url( get_permalink() ) . '">' . $line=$match[0] . ' ...Read more' . '</a>';
```

Comment:
This is not utf-8 nor html safe. Please consider using `wp_html_excerpt()`.
See: http://codex.wordpress.org/Function_Reference/wp_html_excerpt